recipes/python-logstash-async/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "python-logstash-async" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ad68a05aac6c7d2d065b3dd8ba2e2b9bda59d103c75db2aa11b815e03a67c151
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pylogbeat
+    - requests
```

Comment:
Upstream lists `limits`. Any reason not to add it here?

Reply:
> Upstream lists `limits`. Any reason not to add it here?

@dopplershift the problem with `limits` is that it's already hosted in another conda channel (see [https://anaconda.org/equipoise/limits](https://anaconda.org/equipoise/limits)), but I didn't find any option to specify additional channels besides `conda-forge` (which, as I understand, may be by design).

Is there a known way to do that? Or should I just push `limits` into `conda-forge` as well ignoring the fact that it has its own channel? Please advise.

Reply:
You'll need to add `limits` to conda-forge, either in this PR (might need to redo some builds after merging due to issues of which gets built first) or a separate PR. The conda-forge packages do not and cannot depend on packages in any other channel.

Reply:
Please review the pull request for limits ([https://github.com/conda-forge/staged-recipes/pull/17489](https://github.com/conda-forge/staged-recipes/pull/17489) at your convenience.